osfoffline/views/preferences.py

```diff
@@ -197,16 +223,5 @@ def get_remote_top_level_nodes(self):
         except Exception as e:
             logging.warning(e)
 
+        self.finished.emit(remote_top_level_nodes)
```

Comment:
If the server connection fails for any reason at all, the user will not be notified that the fetch failed- the prefs panel will simply show an empty list. We should improve how fetch failures are communicated to the user.